### DISCUSSION TITLE
Tips for your Renovate configuration/usage

### DISCUSSION THREAD

**Discussion by HonkingGoose** [crossplane/crossplane#4132] [Show and tell]
I see that Crossplane is opening some discussions on the Renovate repository. So I got curious and checked your Renovate config file, and how you're using Renovate. I have some tips for you. 😉 

### Pin Dependency Dashboard issue

Pin [^1] your Dependency Dashboard issue, so you don't have to search for it each time you want to look at it.

### Use `description` field instead of comments

This may just be me, but I find the comments in your Renovate config file a bit distracting. You may want to use the `description` [^2] field instead, like this:

```json
{
  "extends": ["config:base", "helpers:pinGitHubActionDigests"],
  "configMigration": true,
  "packageRules": [
    {
      "description": "Wait 3 days before creating a npm update PR",
      "matchDatasources": ["npm"],
      "minimumReleaseAge": "3 days"
    }
  ]
}
```

[^1]: [GitHub Docs, pinning an issue to your repository](https://docs.github.com/en/issues/tracking-your-work-with-issues/pinning-an-issue-to-your-repository)
[^2]: [Renovate docs, `description` field](https://docs.renovatebot.com/configuration-options/#description)

**Comment by jbw976** [comment]
Cool! Thank you @HonkingGoose for taking the time to share your renovate pro tips with us 🙇 

I've pinned the dependency dashboard [issue](https://github.com/crossplane/crossplane/issues/3608) and we'll see how we like that.  It may distract slightly from the other pinned issue we have instructing folks to 👍  on issues they care about, but we'll try it like this and see how we like it!

@phisco may also end up taking a pass through the comments -> description field if he's feeling motivated.  Thanks again! 💪

**Reply by HonkingGoose** [reply]
You can always "unpin" the Dependency Dashboard issue, if needed.

**Comment by phisco** [comment]
Definitely! That was the main reason to use json5 format, but I would definitely prefer having plain json instead!

**Comment by HonkingGoose** [comment]
Oh and another thing. 😄 

### Extend pin GitHub Actions digest preset

You're manually setting `"pinDigests": true` for your GitHub Action dependencies. I think you can use the `helpers:pinGitHubActionDigests` [^1] preset instead:

```json5
  "extends": [
    "config:base",
    "helpers:pinGitHubActionDigests"
  ],
```

Extending the preset avoids duplicating the configuration.

[^1]: [Renovate docs, `helpers:pinGitHubActionDigests` preset](https://docs.renovatebot.com/presets-helpers/#helperspingithubactiondigests)
